app/models/concerns/git_hub_repoable.rb

```diff
@@ -1,32 +1,11 @@
 module GitHubRepoable
   extend ActiveSupport::Concern
 
-  included do
-    before_validation(on: :create) do
-      create_github_repository if organization
-    end
-
-    before_create :add_team_to_github_repository
-    before_create :push_starter_code
-
-    before_destroy :destroy_github_repository
-  end
```

Comment:
I want to keep the methods in the module, but they called differently for `AssignmentRepo` and `GroupAssignmentRepo`